client/src/app/site/agenda/components/agenda-sort/agenda-sort.component.html

```diff
@@ -1,6 +1,12 @@
 <os-head-bar [nav]="false">
     <!-- Title -->
     <div class="title-slot"><h2 translate>Sort agenda</h2></div>
+
+    <!-- Menu to save changes or reset them -->
+    <div class="menu-slot">
+        <button type="button" mat-button (click)="onSave()">{{ 'Save' | translate }}</button>
```

Comment:
Just show both buttons, if something was changed. Hide both after successfull save, so the user gets a feedback, if the saveing was successfull. I can imaging showing a short message in the snack bar on success and error.

Reply:
Same for the call list.